### PR TITLE
Fix the blob encoding type

### PR DESCRIPTION
**Pull request by nazarhussain**
The blobs returned from the beacon-apis are the raw bytes not encoded into ssz. 

Mentioning it as SSZ encoded cause confusion to decode the output from the API e.g. `ssz.Blob.decode(api_response)` which will not match with the actual blobs. 

Some reference: 

https://github.com/ChainSafe/lodestar/blob/ae984f0f6079f78c2445f36ae2ac09c9e868197e/packages/types/src/deneb/types.ts#L8

### CODE REVIEW

**Comment by nazarhussain** [comment]
@g11tech Please take a look at this small change.

**Review by g11tech — APPROVED** [review]
lgtm

**Comment by nflaig** [comment]
> Mentioning it as SSZ encoded cause confusion to decode the output from the API e.g. ssz.Blob.decode(api_response) which will not match with the actual blobs.

What exactly are you trying to do? If you wanna deserialize the raw hex-encoded blob you can do `ssz.Blob.deserialize()`

File: types/primitive.yaml [hunk 1]
```diff
@@ -127,7 +127,7 @@ Blob:
   type: string
   format: hex
   pattern: "^0x[a-fA-F0-9]{262144}$"
-  description: "A blob is `FIELD_ELEMENTS_PER_BLOB * size_of(BLSFieldElement) = 4096 * 32 = 131072` bytes (`DATA`) representing a SSZ-encoded Blob as defined in Deneb"
+  description: "A blob is `FIELD_ELEMENTS_PER_BLOB * size_of(BLSFieldElement) = 4096 * 32 = 131072` bytes (`DATA`) representing a Blob as defined in Deneb"
```
[the remark this answers is not in the document]
Comment (nazarhussain) [reply types/primitive.yaml:130]: It's raw bytes which was sent from the sender of EIP4844 transaction. If we considered it SSZ-Serialized, one will try to deserialize to get the actual blob, which is not the case here. 

The raw bytes sent in transaction are the exact bytes returned over the API, where this description is been used.

**Review by nflaig — APPROVED** [review]
While I don't think the note is necessarily incorrect, doesn't hurt to remove it either

**Pull request merged**